speech/cloud-client/transcribe_stream.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# author: Yogesh Garg
```

Comment:
@yogeshg I think to be accepted you would need to add the Google copyright comment as can be seen [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/3ae5f4691de8d03d87a899fda0b1ebce522371e0/speech/cloud-client/transcribe.py#L3-L15).

Your authorship should be in the commits. I don't see it displayed this way in the other files but @gguuss or @jonparrott would know for sure.